contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,175 @@
+   * @dev Only callable by OCR2DROracles that have been approved on the Registry
+   */
+  function beginBilling(bytes calldata data, RequestBilling calldata billing) external returns (bytes32);
+
+  /**
+   * @notice Finalize billing process for an OCR2DR request
+   * @param requestId contains the proof and response
+   * @param response response data from DON consensus
+   * @param err error from DON consensus
+   * @param transmitter the Oracle who sent the report
+   * @param signers the Oracles who signed the report
+   * @param initialGas the initial amount of gas that was sent by the transmitter when submitting the report
+   * @dev Only callable by OCR2DROracles that have been approved on the Registry
+   * @dev simulated offchain to determine if sufficient balance is present to fulfill the request
+   */
+  function concludeBilling(
+    bytes32 requestId,
+    bytes calldata response,
+    bytes calldata err,
+    address transmitter,
+    address[31] memory signers, // Matches maxNumOracles from OCR2Abstract.sol
+    uint32 initialGas
+  ) external returns (uint96);
+
+  /**
+   * @notice Get request commitment
+   * @param requestId id of request
+   * @dev used to determine if a request is fulfilled or not
+   */
+  function getCommitment(bytes32 requestId)
+    external
+    view
+    returns (
+      address,
+      uint64,
+      uint32
+    );
+
+  /**
+   * @notice Create a new subscription.
+   * @return subId - A unique subscription id.
+   * @dev You can manage the consumer set dynamically with addConsumer/removeConsumer.
+   * @dev Note to fund the subscription, use transferAndCall. For example
+   * @dev  LINKTOKEN.transferAndCall(
+   * @dev    address(REGISTRY),
+   * @dev    amount,
+   * @dev    abi.encode(subId));
+   */
+  function createSubscription() external returns (uint64 subId);
+
+  /**
+   * @notice Get details about a subscription.
+   * @param subId - ID of the subscription
+   * @return balance - LINK balance of the subscription in juels.
+   * @return owner - owner of the subscription.
+   * @return consumers - list of consumer address which are able to use this subscription.
+   */
+  function getSubscription(uint64 subId)
+    external
+    view
+    returns (
+      uint96 balance,
+      address owner,
+      address[] memory consumers
+    );
+
+  /**
+   * @notice Request subscription owner transfer.
+   * @param subId - ID of the subscription
+   * @param newOwner - proposed new owner of the subscription
+   */
+  function requestSubscriptionOwnerTransfer(uint64 subId, address newOwner) external;
+
+  /**
+   * @notice Request subscription owner transfer.
+   * @param subId - ID of the subscription
+   * @dev will revert if original owner of subId has
+   * not requested that msg.sender become the new owner.
+   */
+  function acceptSubscriptionOwnerTransfer(uint64 subId) external;
```

Comment:
I am not 100% sure of the reason for needing to request & accept ownership vs the owner just transferring to a new address. Is it for verfication purposes? (ie: in case owner screwed up and sent ownerhip the wrong address) or some other reason?

Reply:
This was explained in a comment in the OCR2DRRegistry contract.

Reply:
Not too sure myself, just inherited it from VRFv2.
Seems like CLL rolled their own ownership contracts. [OpenZeppelin's Ownable.sol](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol) for example does not have the accepting step.

I think it would be for security reasons. Like say I was doing bad things and knew I was about to go court. So months before I transfer ownership to you. Now in court I say, "Look Morgan was the owner, I don't have anything to do with this".
Maybe that is reaching haha.

Reply:
https://github.com/smartcontractkit/chainlink/pull/7158/files#diff-09426f353874aaed689b8885a83dfb82ceb85879b2637d203eb8d4f4288ac924R217